packages/common/src/utils/SafeLocalStorage.ts

```diff
@@ -1,25 +1,21 @@
-import type { CaipNetwork } from './TypeUtil.js'
-
 export type SafeLocalStorageItems = {
   '@appkit/wallet_id': string
   '@appkit/wallet_name': string
   '@appkit/solana_wallet': string
   '@appkit/solana_caip_chain': string
-  '@appkit/active_caip_network': CaipNetwork
   '@appkit/active_caip_network_id': string
   '@appkit/connected_connector': string
   '@appkit/connected_social': string
   '@appkit/connected_social_username': string
   '@appkit/recent_wallets': string
-  '@appkit/deeplink_choice': { href: string; name: string }
```

Comment:
Is this ok?

Reply:
Yeap, when we read the `appkit/deeplink_choice` value, we only use `href`, not the `name` - and since we just switched to `@appkit/` prefix, this should be good